src/src/AdminBundle/Entity/EventType.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace AdminBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * EventType
+ *
+ * @ORM\Table(name="event_type")
+ * @ORM\Entity(repositoryClass="AdminBundle\Repository\EventTypeRepository")
+ */
+class EventType extends BaseEntity
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="slug", type="string", length=50)
+     * @Assert\NotBlank()
+     */
+    private $slug;
```

Comment:
sprav nech je to unique. tak ako tu https://github.com/TIS2017/Nobelisti/pull/64/files

Reply:
Oki